### DISCUSSION TITLE
Fullscreen Mode

### DISCUSSION THREAD

**Discussion by GoTeamScotch** [SVG-Edit/svgedit#547] [New feature request]
SVG Edit could use a fullscreen button. I have this working in my development copy of the app, complete with icons as needed. I copied the existing code formatting where possible and put the icon right after the Undo button in the top toolbar.

My question is this: is it ok to include [screenfull](https://github.com/sindresorhus/screenfull.js) in SVG Edit? It's covered under an MIT license.

If this is ok, I can submit a pull request later.

**Comment by codedread** [comment]
I'll be honest, I've forgotten the legalities of SVG-edit... Several of the third-party libraries we use are MIT licensed though, so I think this is ok.

**Comment by GoTeamScotch** [comment]
Ok I will need to fork the repo and put in my edits from a clean copy of the base branch. The copy I have is too modified to upload changes directly.

**Comment by belonesox** [comment]
@GoTeamScotch, have you any success with it?

**Comment by moeenio** [comment]
@GoTeamScotch ?
